GovUkDesignSystem/GovUkDesignSystemComponents/DateInput.cshtml

```diff
@@ -2,9 +2,20 @@
 @model GovUkDesignSystem.GovUkDesignSystemComponents.DateInputViewModel
 
 @{
-    var id = Model.Id;
-    var errorId = Model.ErrorMessage != null ? id + "-error" : null;
-    var hintId = Model.Hint != null ? id + "-hint" : null;
+    string describedBy = Model.Fieldset?.DescribedBy;
```

Comment:
If I'm reading this and the way the tests are written correctly, we don't allow this to be null but the errorId or hintId to not be null. Is that right?

Reply:
AFAIK any or all of them might be null (and this should still work)

Reply:
Ah, you are right, looks like it converts nulls into empty strings during concatenation. Carry on then :)